### DISCUSSION TITLE
Getting nice byte coloring like in Hex Edit Pro?

### DISCUSSION THREAD

**Discussion by aybe** [WerWolv/ImHex#1426] [Pattern Language Q&A] [answered]
In the defunct Hex Edit Pro software, there is a nice feature of coloring bytes according their value:

<img width="636" alt="image" src="https://github.com/WerWolv/ImHex/assets/1537591/e8c439aa-c03c-4cdf-88f7-c85404031ad9">

Basically, it colors a specific range of values:

<img width="406" alt="image" src="https://github.com/WerWolv/ImHex/assets/1537591/44b383e6-f1b6-4d86-afc8-e7c810fcfb5d">

Is there any (easy) way to achieve the same with ImHex? 🎯

**Comment by WerWolv** [comment, marked as answer]
This has been implemented in v1.32.0 now called `Highlighting Rules`. They can be found under the `Edit` menu :)

**Reply by aybe** [reply]
Great! I've started using it and going to report few issues about it (usability, improvements). Thanks 🥳
